core/src/main/java/com/kin/ecosystem/core/util/ErrorUtil.java

```diff
@@ -39,31 +38,32 @@
 	public static final int ERROR_CODE_REQUEST_TIMEOUT = 408;
 	public static final int ERROR_CODE_CONFLICT = 409;
 	public static final int ERROR_CODE_INTERNAL_SERVER_ERROR = 500;
+	public static final int ERROR_CODE_TRANSACTION_FAILED_ERROR = 700;
 
-	public static final int ERROR_CODE_USER_NOT_FOUND = 4046;
```

Comment:
we did, but it's part of `ERROR_CODE_NOT_FOUND` so we handled it specifically but I'm taking the message from the `apiException` so it will be a more clear message and not default one.